sxg_rs/src/headers.rs

```diff
@@ -124,34 +127,90 @@ impl Headers {
         }
         Ok(())
     }
-    pub fn get_signed_headers_bytes(&self, status_code: u16, mice_digest: &[u8]) -> Vec<u8> {
-        use crate::cbor::DataItem;
+    // Filters the link header to comply with
+    // https://github.com/google/webpackager/blob/main/docs/cache_requirements.md.
+    fn process_link_header(value: &str, fallback_url: &Url) -> String {
+        static ALLOWED_PARAM: Lazy<HashSet<&'static str>> = Lazy::new(|| {
```

Comment:
Excuse my naive question.  Why did you decide to define statics inside the function as opposed to using const or https://crates.io/crates/phf?

Reply:
The hash set needs run-time heap allocation so it can't be a const value. Both `once_cell::Lazy` and `phf` give us static variables. We selected `Lazy` because [it will be part of Rust std in the future](https://doc.rust-lang.org/std/lazy/struct.Lazy.html).